blog/_posts/2021-03-15-scaling-inlets.md

```diff
@@ -0,0 +1,349 @@
+---
+layout: post
+title: Scaling inlets like a Pro with the help of Kubernetes
```

Comment:
`Scaling inlets like a Pro with the help of Kubernetes`

I wonder if adding a unit would make this a stronger post?

`Learn how to scale to hundreds of inlets tunnels with Kubernetes`

Then in the conclusion, you could mention how inlets-cloud can make it easier to go to even bigger numbers like thousands.

See if you can get some ideas from:

https://neilpatel.com/blog/the-step-by-step-guide-to-writing-powerful-headlines/